modules/overview/partials/proc_installing-che-using-the-che-operator-in-openshift-4-web-console.adoc

```diff
@@ -57,7 +45,7 @@ Operators are a method of packaging, deploying, and managing a Kubernetes applic
 . In the *Operator Details* screen, in the *Details* tab, inside of the *Provided APIs* section, click the *Create Instance* link.
 
 . The *Create CheCluster* page contains the configuration of the overall {prod-short} instance to create. It is the `CheCluster` Custom Resource. For an installation using the default configuration, keep the default values.
-To modify the configuration, see xref:installation-guide:configuring-the-che-installation.adoc[].
+To modify the configuration, see xref:configuring-the-{prod-id-short}-installation_crw[Configuring the {prod-short} installation].
```

Comment:
It is in another guide (installation-guide v. overview) so `xref:installation-guide:configuring-the-che-installation.adoc[]` is correct.

Reply:
Ack, reverted.